glean-core/ios/GleanTests/Metrics/UrlMetricTests.swift

```diff
@@ -0,0 +1,123 @@
+/* This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this
+ * file, You can obtain one at http://mozilla.org/MPL/2.0/. */
+
+@testable import Glean
+import XCTest
+
+// swiftlint:disable force_cast
+// REASON: Used in a test
+class UrlMetricTypeTests: XCTestCase {
+    override func setUp() {
+        Glean.shared.resetGlean(clearStores: true)
```

Comment:
This has the potential to leak pings, so one of us needs to remember to fix it depending on who's PR lands first ;)

Reply:
Landing this now, can you adopt that in your PR?